src/headers/SiteHeader.tsx

```diff
@@ -439,11 +438,16 @@ const SiteHeader = (props: SiteHeaderProps) => {
 
   return (
     <header className={"site-header"}>
+      {props.mainContentId && (
+        <a className="skip-link" href={`#${props.mainContentId}`}>
+          Skip to main content
```

Comment:
Doesn't this need to be a translated string?

Reply:
Yes! Forgot to change that back. Thank you for the catch.